packages/jazz-tools/src/tests/subscribe.test.ts

```diff
@@ -286,7 +286,6 @@ describe("subscribeToCoValue", () => {
     const lastValue = updateFn.mock.lastCall[0];
     expect(lastValue).not.toBe(initialValue);
     expect(lastValue.name).toBe("Me and Luigi");
-    expect(initialValue.name).toBe("General");
```

Comment:
Keeping the content ref breaks this check, because initialValue content is the same as lastValue.

This can be an issue only with React Concurrent, but we have this problem anyway (see #565)

This issue can be solved once we land #714